core/storage/opportunity/gae_models.py

```diff
@@ -52,6 +52,18 @@ def get_deletion_policy():
         """
         return base_models.DELETION_POLICY.KEEP_IF_PUBLIC
 
+    @classmethod
+    def has_reference_to_user_id(cls, unused_user_id):
+        """ExplorationOpportunitySummaryModel doesn't reference any user_id
+        directly.
+        Args:
```

Comment:
Insert single newline above this one.

/cc @kevinlee12 @oppia/dev-workflow-team Please consider this as a possible lint check? It seems to be occurring rather frequently among various PRs.

Reply:
Done.

Reply:
I've filed https://github.com/oppia/oppia/issues/7868